muxers/yamux/src/lib.rs

```diff
@@ -252,62 +283,69 @@ impl WindowUpdateMode {
     /// > **Note**: With this strategy, there is usually no point in the
     /// > receive buffer being larger than the window size.
     pub fn on_read() -> Self {
-        WindowUpdateMode(yamux::WindowUpdateMode::OnRead)
+        WindowUpdateMode(yamux012::WindowUpdateMode::OnRead)
     }
 }
 
 impl Config {
     /// Creates a new `YamuxConfig` in client mode, regardless of whether
     /// it will be used for an inbound or outbound upgrade.
+    #[deprecated(note = "Will be removed with the next breaking release.")]
```

Comment:
Are we just deprecating this because users shouldn't need it?

Reply:
Yes. I don't know why `client` and `server` exist. In other words, I am not aware of a use-case. Are you @thomaseizinger?